### PR TITLE
feat: add airdrop faucet tokens description in readme 

### PR DESCRIPTION
**Pull request by jorgeantonio21**
Description
---
Adds a description on how to obtain faucet tokens in the README.

Motivation and Context
---

How Has This Been Tested?
---

What process can a PR reviewer use to test or verify this change?
---

Breaking Changes
---

- [x] None
- [ ] Requires data directory to be deleted
- [ ] Other - Please specify

### CODE REVIEW

**Review by stringhandler — COMMENTED** [review]
I prefer to have both options in the readme, with a (Recommended) for the claim free funds

File: README.md [hunk 1]
```diff
@@ -97,45 +97,74 @@ cargo run --bin tari_validator_node_cli -- templates publish
...
+above).
+
 ### Claiming L1 burn Tari on the DAN
 
-The user will need to claim burn tari (from the L1) before submitting transactions, otherwise, it will not be
-able to pay for network fees. To be able to claim burn tari, it is necessary to first burn L1 tari, on the base layer. After
+On the other hand, a user wishing to pay for fees in the Tari network, can claim these from MinoTari on the base layer.
+This process is more complex and tedious at the moment, but has the advantage that it will be process to be used when
+the mainnet is launched. To be able to claim burn Tari, it is necessary to first burn MinoTari, on the base layer. After
 burning tari on the Tari base layer, the client is prompted with the following data
```
Comment (sdbondi) [inline README.md]:
```suggestion
L1 Minotari coins and able to be burnt and claimed, they may convert (1:1) these to Tari coins on the layer-2 network. This process is fairly complex and manual at the time of writing, but is planned to improve in the future. The first step is to burn Minotari base layer funds making sure to include a claim public key. A private claim key must be known to claim the funds on the layer-2 Tari network. Burning yields the following data:

```

File: README.md [hunk 2]
```diff
@@ -97,45 +97,74 @@ cargo run --bin tari_validator_node_cli -- templates publish
...
-    "rangeproof": <rangeproof>
+    "commitment": <COMMITMENT>,
+    "ownership_proof": <OWNERSHIP_PROOF>,
+    "rangeproof": <RANGEPROOF>
 }
 ```
 
-Now, it is possible to claim burn Tari on the DAN layer, as follows. Create a new `.json` file, with path
-`<json_file_to_retrieve_burn_tari>`
+Notice that, to burn MinoTari, the user had to pass a claim public key
```
Comment (sdbondi) [inline README.md]: Maybe not needed with the above wording

File: README.md [hunk 3]
```diff
@@ -97,45 +97,74 @@ cargo run --bin tari_validator_node_cli -- templates publish
...
-}
+"claim_public_key": <CLAIM_PUBLIC_KEY>
+```
+
+As in the previous section, the user creates an account on the Tari network, via the wallet cli. After an account is created
+with 
+
+```
+"account_name": <ACCOUNT_NAME>
 ```
```
Comment (sdbondi) [inline README.md]:
```suggestion
The user must create an account on the Tari network using the `accounts create` wallet CLI command prior to claiming. See the previous section for details on creating an account. The public key of this account should be used as the claim public key when burning funds.
```

File: README.md [hunk 4]
```diff
@@ -97,45 +97,74 @@ cargo run --bin tari_validator_node_cli -- templates publish
 
 Once the template is registered on the base layer and sufficiently mined, you should see it in the `templates` table of the `global_storage.sqlite` file  under `data/validator`. The `compiled_code` column should contain binary data and the `status` column should be `Active`.
 
+### Get airdropped base layer (Mino)Tari tokens to pay for fees
+
+Before the user can start submitting transactions to the network, it has to obtain base layer (Mino)Tari tokens. At the moment, the
+simplest way to do so is to request an airdrop of (free test) tokens from the network itself. In order to do so, the user should
+first create an account, using the tari wallet client, as follows:
+
```
Comment (sdbondi) [inline README.md]:
```suggestion
Before the user can start submitting transactions to the network, it has to obtain base layer (Mino)Tari tokens to pay for fees. At the moment, the
simplest way to do so is to request an airdrop of (free test) tokens from the network itself. In order to do so, the user should
first create an account, using the tari wallet client, as follows:

```

**Review by sdbondi — COMMENTED** [review]
Made some wording changes, feel free to use/throw away as you see fit :)

File: README.md (hunk 1 above)
Comment (jorgeantonio21) [reply README.md]: Addressed

**Review by sdbondi — APPROVED** [review]

**Pull request merged**